tests/Console/ModelsCommand/CustomCollection/Test.php

```diff
@@ -6,9 +6,6 @@
 
 use Barryvdh\LaravelIdeHelper\Console\ModelsCommand;
 use Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\AbstractModelsCommand;
-use Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\CustomCollection\Collections\SimpleCollection;
-use Illuminate\Filesystem\Filesystem;
-use Mockery;
```

Comment:
Lots of them were leftovers from https://github.com/barryvdh/laravel-ide-helper/pull/1019